app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -110,7 +110,7 @@ echo "#"
 <% if plugin_present?('katello') -%>
 if [ -f /etc/redhat-release ]; then
     register_katello_host(){
-        UUID=$(subscription-manager identity | head -1 | awk '{print $3}')
+        UUID=$(subscription-manager identity | head -1 | awk -F':' '{gsub(" ","",$2);print $2}')
```

Comment:
Couldn't we use `: ` (colon space) as a field separator instead of spliting on a colon and the trimming the space?